openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -509,6 +509,64 @@ def _default_settings(cls):
             output_settings=MultiStateOutputSettings(),
         )
 
+    @classmethod
+    def _adaptive_settings(
+            cls,
+            stateA: ChemicalSystem,
+            stateB: ChemicalSystem,
+            mapping: gufe.LigandAtomMapping | list[gufe.LigandAtomMapping]
+    ) -> RelativeHybridTopologyProtocolSettings:
+        """
+        Get the recommended OpenFE settings for this protocol based on the input states involved in the
+        transformation.
+
+        These are intended as a suitable starting point for creating an instance of this protocol, which can be further
+        customized before performing a Protocol.
+
+        Notes
+        -----
+            The fast settings used are still being refined and tested and will only be applied
+            to net neutral transformations.
+        """
+        # get an unfrozen copy of the default settings that we can edit
+        protocol_settings = cls.default_settings().unfrozen_copy()
+
+        if mapping.get_alchemical_charge_difference() != 0:
+            # apply the recommended charge change settings taken from the industry benchmarking as fast settings not validated
+            # <https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/blob/2df362306e2727321d55d16e06919559338c4250/industry_benchmarks/utils/plan_rbfe_network.py#L128-L146>
+            wmsg = ("Charge changing transformation between ligands "
+                    f"{mapping.componentA.name} and {mapping.componentB.name}. "
+                    "A more expensive protocol with 22 lambda windows, sampled "
+                    "for 20 ns each, will be used here.")
+            warnings.warn(wmsg)
```

Comment:
Thoughts on warning here but not warning on the default behaviour (below)? I.e. should this be just at INFO level since we expect it to be adaptive?

Reply:
This was to maintain the current CLI experience, where this warning is displayed for charge change edges. However, more generally, perhaps an INFO-level log would be better for each edge, indicating if the settings are adapted from the defaults.